pkg/state/state.go

```diff
@@ -961,6 +966,11 @@ func (st *HelmState) PrepareCharts(helm helmexec.Interface, dir string, concurre
 	}
 
 	var builds []*chartPrepareResult
+	pullChan := make(chan *PullCommand)
+	defer func() {
+		pullChan <- nil
```

Comment:
I believe that we can just `close(pullChan)` here and change the for-loop to:

```
for pullCmd := range pullChan {
	err := helm.ChartPull(pullCmd.ChartRef)
	pullCmd.responseChan <- err
}
```

for simplicity. I'd appreciate it if you could try that that submit another pull request for cleanup purpose 😃

Reply:
Okay, I didn't know that (I'm a noob in Go language :smile:  ), I'll do a PR for that !